app/src/main/res/layout/nav_header_navigation_drawer.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="@dimen/nav_header_height"
```

Comment:
If you are not using this anywhere you can directly specify height.

Reply:
Done